### ISSUE TITLE
Bareword STDIN fails with -t

### ISSUE THREAD

**Issue by scottchiefbaker** [xsawyerx/guacamole#88]
I'm testing my code against `use standard` and I found what I think is a discrepancy. I have code that tests if STDIN is an interactive terminal thusly:

    if (-t STDIN == 0) { ...

This fails with `standard`, but I think it should be allowed according to what I've read. Alternately I did find that:

    if (-t *STDIN == 0) { ...

removes the warning, but I'm not sure it should be required per the documentation.

**Comment by xsawyerx** [comment]
Until we fix this, you can write `if ( -t STDIN() )` - that works in Perl and Guacamole understands it.

**Comment by mfontani** [comment]
```bash
$ perl -Mstandard -wc <( echo 'if (-t STDIN()) { print "hi" }' )
/proc/self/fd/12 syntax OK
```

and:

```bash
$ perl -Mstandard -wc <( echo 'unless (-t STDIN()) { print "hi" }' )
/proc/self/fd/12 syntax OK
```

... which is fine and expected, but what about this:

```bash
$ perl -Mstandard -wc <( echo 'if (! -t STDIN()) { print "hi" }' )
File '/proc/self/fd/12' does not pass Standard Perl.
Parser says:
> Error in SLIF parse: No lexeme found at line 1, column 10
> * String before error: if (! -t\s
> * The error was at line 1, column 10, and at character 0x0053 'S', ...
> * here: STDIN()) { print "hi" }
> Marpa::R2 exception at /home/marco/perl5/lib/perl5/Guacamole.pm line 2049.
> 
> Failed to parse past: t (char 4, length 1), expected LParen,OpArrow,PackageSep at /home/marco/perl5/lib/perl5/Guacamole.pm line 2065.
/proc/self/fd/12 syntax OK
```

Does one _have_ to use `unless`? :/

**Comment by xsawyerx** [comment]
This is weird. We did have a bug with `!` and requiring parenthesis. #64 but @vickenty fixed it.

Considering the expected input, maybe it's thinking that `-t` is a unary `-` and a subname `t`. This explains it expects parens, arrows, or additional package separators. If so, we can probably fix this with priority.

**Comment by scottchiefbaker** [comment]
This is issue is resolved after testing with 0.005.
